### ISSUE TITLE
Vscode Extensions?

### ISSUE THREAD

**Issue by RenjiYuusei** [Dror-Bengal/CursorFocus#5]
I think if we make it into vscode extensions it will be more convenient and compact

**Comment by Dror-Bengal** [comment]
Yes agree, but I believe we should stabilize it a bit more. We should also provide more support for better descriptions for different languages. Additionally, we could create a development roadmap with tracking.

**Comment by RenjiYuusei** [comment]
> Đúng, đồng ý, nhưng tôi tin chúng ta nên ổn định nó thêm một chút. Chúng tôi cũng nên cung cấp nhiều hỗ trợ hơn để mô tả tốt hơn cho các ngôn ngữ khác nhau. Ngoài ra, chúng tôi có thể tạo lộ trình phát triển có tính năng theo dõi.

do you have discord so we can talk easily

**Comment by Dror-Bengal** [comment]
https://discord.gg/PThnpph9
